src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
CJMCU undefines SERIAL_RX but SPEKTRUM_BIND is still defined. Without this condition the error still ccurs. Or did missing anything?